Assets/Scripts/ViconNexusUnityStream/CustomSubjectScript.cs

```diff
@@ -127,19 +119,19 @@ protected void SetupMessagePack()
 
         protected void LateUpdate()
         {
-            if (useDefaultData)
+            if (subjectDataManager.UseDefaultData)
             {
                 ProcessData(defaultDataObj, defaultData);
             }
             else
             {
-                ProcessData(dataStreamer.StreamedData[subjectName], dataStreamer.StreamedRawData[subjectName]);
+                ProcessData(subjectDataManager.StreamedData[subjectName], subjectDataManager.StreamedRawData[subjectName]);
```

Comment:
Maybe this if-else logic should be managed internally within subjectDataManager. CustomSubject should remain unaware of the default data status.

Reply:
That also requires rewriting how the data gets written out (another TODO). We'll circle back to that when we know this works.